src/Cli/dotnet/Parser.cs

```diff
@@ -162,9 +165,7 @@ public static bool TokenPerLine(string tokenToReplace, out IReadOnlyList<string>
         public static CliConfiguration Instance { get; } = new(ConfigureCommandLine(RootCommand))
         {
             EnableDefaultExceptionHandler = false,
-            EnableParseErrorReporting = true,
```

Comment:
Why do we remove the error reporting here and in the other place? S.CL uses this for reporting parse errors in a nice way - I'm actually pretty confused why our tests pass with such a large change!

Reply:
This feature was changed and this property no longer exists on this class. There is a discussion [here](https://github.com/dotnet/command-line-api/pull/2205/files#r1221559772) that you responded to. This concept is bound to the `ParseErrorAction` now. From my understanding, it isn't a thing you turn on/off, but the type of action returned when a parsing error occurs. It is actually one of the things I've made a PR for, as there was a problem with a stray newline in its output: https://github.com/dotnet/command-line-api/pull/2289

Reply:
Wow I completely forgot about both of those things 😅

Thank you for the reminder, and the explanation! All good from my end now.